databricks/koalas/window.py

```diff
@@ -49,6 +55,22 @@ def __getattr__(self, item: str) -> Any:
 
 
 class Expanding(_RollingAndExpanding):
+    def __init__(self, kdf_or_kser, min_periods=1):
+        from databricks.koalas import DataFrame, Series
+        from databricks.koalas.groupby import SeriesGroupBy, DataFrameGroupBy
+
+        if min_periods < 0:
+            raise ValueError("min_periods must be >= 0")
+        self._min_periods = min_periods
+        self.kdf_or_kser = kdf_or_kser
+        if not isinstance(kdf_or_kser, (DataFrame, Series, DataFrameGroupBy, SeriesGroupBy)):
```

Comment:
I'll fix this one nicer when I add `count` into `DataFrameGroupBy` and `SeriesGroupBy` in a separate PR.